lesson07/assignment/hello-world/src/components/ComparisonPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div>
+    <header>
+      <h1>Comparison of Coffee Cultures</h1>
+        <img src="images/top-view-tasty-coffee-with-coffee-beans.jpg" alt="top view coffee with coffee beans">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <div class="container">
+              <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent">
+                <ul class="navbar-nav">
+                    <li class="nav-item"><router-link to="/src/components/HomePage">Home</router-link></li>
+                    <li class="nav-item"><router-link to="/src/components/HistoryPage">Images</router-link></li>
+                    <li class="nav-item"><router-link to="/src/components/DrinksPage">Grid System</router-link></li>
+                </ul>
+              </div>
+            </div>
+          </nav>
+    </header>
+    <main>
+        <aside>
+            <h2>Coffee Cultures</h2>
+            <p>Now that you know a little more about the origin of coffee, you can now explore the different coffee cultures around the world.</p>
+        </aside>
+        <section class="content">
+            <h2>United States</h2>
+            <p>Coffee in America is about convenience, accessibility, and speed. You will typically find Americans drinking coffee at home or grabbing a cup quickly before heading into the office to begin their workday. Although the craft coffee culture has very much taken off in America, even those craft cocktails are often taken to go.</p>
+            <p>This is begining to change with the third wave of coffee. See more about the three waves of coffee below.</p>
+            <div id="coffeeWavesAccordion">
```

Comment:
You are missing the class="accordion" on this div.  Without it, you don't get the styles from bootstrap.

`<div id="coffeeWavesAccordion" class="accordion">`